config/dev.webpack.config.ts

```diff
@@ -21,8 +20,8 @@ const { config: webpackConfig, plugins } = config({
     https: true,
     useFileHash: false,
```

Comment:
We can and should also remove both `https` and `useFileHash`, both will be handled by the shard config.

Reply:
Done. I would restrain from big changes in [dev/prod].webpack.config.js in this PR since it's not the main goal. We could revisit all of the options once we do migration to fec.config.js